veramo-sdk-for-cheqd/verifiable-credentials/verify-jwt-vc.md

```diff
@@ -48,18 +48,17 @@ If you belive you did everything right(created a verifiable credential and you d
 Credential could not be verified
 ```
 
-#### You can fix it this way:
+#### You can fix it the following way
 
-1. First, make sure you are using a `node` with version `16` or above.
-2. Second, Change veramo cli version to `@veramo/cli@3.1.6-next.170 -g` as previous versions cause few issues in verification.
+- First, make sure you are using a `node` with version `16` or above.
```

Comment:
Instead of mentioning this in multiple documents, Just mentioning at the [start](https://github.com/cheqd/identity-docs/blob/DEV-1735/veramo-sdk-for-cheqd/setup-cli.md#11-install-veramo-cli) will be sufficient right? The same applies for @veramo/cli versions as well

Reply:
Nice catch. Yep, I actually added it at start of doc but for some reason I kept it here as well.

Reply:
But, since it's a troubleshooting guide it might help some frustrated devs notice it. What do you say?